README.md

```diff
@@ -2,55 +2,158 @@
+
+```swift
+let asManyApplesAsOranges = false
+let notAsManyApplesAsOranges = !asManyApplesAsOranges // true
+```
+
+It can be read as *not* `asManyApplesAsOranges` to make its use as a prefix clearer. If we choose our variable names carefully, we can ensure that such logical expressions can be read like this and make our code more readable in that way.
+
+The logical `AND` operator forms a logical expression which is `true` if both values are `true`.
 
 ```swift
 let isTheSkyBlue = true
 let isTheGrassGreen = true
 let canHumansFly = false
 let canDolphinsWalk = false
+
+let isTheSkyBlueAndTheGrassGreen = isTheSkyBlue && isTheGrassGreen // true, because both values are true
+let canHumansFlyAndIsTheSkyBlue = canHumansFly && isTheSkyBlue // false, because canHumansFly is false
+let canHumansFlyAndDolphinsWalk = canHumansFly && canDolphinsWalk // false, because both values are false
 ```
 
-* Explain and use all the various comparison operators:
-	* Equal to (a == b)
-	* Not equal to (a != b)
-	* Greater than (a > b)
-	* Less than (a < b)
-	* Greater than or equal to (a >= b)
-	* Less than or equal to (a <= b)
-	
+Since the statement will be `false` if a single value is `false`, an expression `canHumansFlyAndIsTheSkyBlue` will stop evaluating after checking `canHumansFly`, because that is already `false`. This method is called *short circuiting*. If the second value is the result of a function, that function would not even be called:
+
+```swift
+func printIfCalled() -> Bool {
+	print("Function is called.")
+	return true
+}
+
+let fullEvaluation = true && printIfCalled() // Will print "Function is called."
+let shortCircuit = false && printIfCalled() // Nothing will be printed, because of short circuiting
+```
 
-* Explain and use all of the various logical operators:
-	* Logical NOT (!a)
-	* Logical AND (a && b)
-	* Logical OR (a || b)
+In contrast to that, the logical `OR` operator will evaluate to `true` if at least one of the values is `true`:
 
 ```swift
-1 == 1   // true because 1 is equal to 1
-2 != 1   // true because 2 is not equal to 1
-2 > 1    // true because 2 is greater than 1
-1 < 2    // true because 1 is less than 2
-1 >= 1   // true because 1 is greater than or equal to 1
-2 <= 1   // false because 2 is not less than or equal to 1
+let isTheSkyBlue = true
+let isTheGrassGreen = true
+let canHumansFly = false
+let canDolphinsWalk = false
+
+let isTheSkyBlueOrTheGrassGreen = isTheSkyBlue || isTheGrassGreen // true, because both values are true
+let canHumansFlyOrIsTheSkyBlue = canHumansFly || isTheSkyBlue // true, because isTheSkyBlue is true
+let canHumansFlyOrDolphinsWalk = canHumansFly || canDolphinsWalk // false, because both values are false
 ```
 
```

Comment:
Similar to above, taking out the variables:
```swift
isTheSkyBlue || isTheGrassGreen // true, because both values are true
canHumansFly || isTheSkyBlue // true, because isTheSkyBlue is true
canHumansFly || canDolphinsWalk // false, because both values are false
```